.github/workflows/deploy.yml

```diff
@@ -17,17 +18,21 @@ on:
 
 jobs:
   deploy:
+    # do not run the sheduled run on forks
+    if: github.event != 'schedule' || github.repository_owner == 'Cockatrice'
+
     runs-on: ubuntu-latest
 
     env:
+      DEPLOY: ${{github.ref == 'refs/heads/master'}}
```

Comment:
This expression should always be true as we only run on pr/push triggers on master?
Therefore the `Checkout output branch` step is always run as well.

Would those expressions work instead and return "true" only when we want to deploy?
${{ github.event_name != 'pull_request' }}
${{ github.base_ref != 'refs/heads/master' }}

Reply:
the really curious case is why this check is correct when done directly in the if statement and not in the environment variable

Reply:
it's because it is 'false' and not false https://github.com/Cockatrice/Magic-Spoiler/pull/260/checks?check_run_id=1500055631#step:3:14